spec/types/make-brainstem-type-spec.js

```diff
@@ -114,10 +114,21 @@ describe('makeBrainstemType', () => {
     });
 
     it('forwards the request to brainstem-redux', () => {
-      const options = 'OPTIONS';
+      const options = { stuff: 'options' };
```

Comment:
This teaches me a lesson to use stubbed data objects that are similar to the real life object.

Reply:
Juanca = :man_student: 
- Bobby